docs(multi-input): remove using whenDefined and improve docs (#315)
Refinitiv/refinitiv-ui: documents/src/pages/elements/multi-input.md

```
```javascript
const el = document.getElemetnById('multi-input');
const result = el.add({
  value: 'newItemValue',
  label: 'New Item'
});

console.log(result); // newly added item or null if new item is invalid
```

`ef-multi-input` provides `item-added` event. The event will fire when users add a new item to the control by pressing the enter key on their keyboard. In the event object, you can access `item` for an item object that is newly added and `items` for an array of items currently shown in the control.

For example, if a user types `Mango` and presses the enter key in `ef-multi-input`.

```javascript
el.addEventListener('item-added', (event) => {
  console.log(event.detail.item) // Mango object.

>   console.log(event.detail.items) // Array of item in control before mango is added.

Reviewer: ```suggestion
  console.log(event.detail.items) // Array of items in control before mango is added.
```

});
```

The event can be cancelled so you can add custom logic to prevent some items from being added to the control.

```javascript
el.addEventListener('item-added', (event) => {
  if (event.detail.item.value === 'Mango') {

>     event.preventDefault(); // Prevent  Mango to be added to the list

Reviewer: ```suggestion
    event.preventDefault(); // Prevent Mango from adding to the list
```
